src/number_theory/bernoulli.lean

```diff
@@ -226,6 +240,9 @@ end
 /-- The Bernoulli numbers are defined to be `bernoulli'` with a parity sign. -/
 def bernoulli (n : ℕ) : ℚ := (-1)^n * (bernoulli' n)
 
+lemma bernoulli'_eq_neg_one_pow_mul_bernoulli (n : ℕ) : bernoulli' n = (-1)^n * bernoulli n :=
```

Comment:
Concerning the suggestion to change this name. I agree that this name is a bit long, but we don't want confusion with the other lemma. How about this:

* rename `bernoulli_eq_bernoulli'` to `bernoulli_eq_bernoulli'_of_ne_one`
* rename this lemma to `bernoulli'_eq_bernoulli`

That way both names are reasonably short, and the there shouldn't be any confusion.

Reply:
Done.
Thanks for the suggestion: that make sense!